Source/Core/Core/HW/WiiSaveCrypted.cpp

```diff
@@ -337,8 +337,7 @@ void CWiiSaveCrypted::ImportWiiSaveFiles()
     }
     else
     {
-      std::string filename =
-          Common::EscapeFileName(reinterpret_cast<const char*>(file_hdr_tmp.name));
+      std::string filename = Common::EscapePath(reinterpret_cast<const char*>(file_hdr_tmp.name));
```

Comment:
This doesn't make sense to my brain, being called *filename* and being concatenated to a full path in the next line. Can you explain why (is it perhaps a relative path, not just a file name) and potentially even add a comment?